client/cli/src/config.rs

```diff
@@ -346,6 +349,11 @@ pub trait CliConfiguration<DCV: DefaultConfigurationValues = ()>: Sized {
 		Ok(RPC_DEFAULT_MAX_SUBS_PER_CONN)
 	}
 
+	/// Get maximum number of subscriptions per connection.
```

Comment:
Is this only subscriptions? or messages generally?

maybe:

```suggestion
	/// The number of messages the RPC server is allowed to keep in memory per connection.
```

Reply:
messages, thanks

bad copy-paste nit